web/thesauruses/cplusplus/control_structures.json

```diff
@@ -0,0 +1,114 @@
+{
+  "meta": {
+    "language": "language_id",
+    "language_version": "version.number",
+    "language_name": "Human-Friendly Language Name"
+  },
+  "categories": {
+    "Conditional Statements (ifs)": [
+      "if_conditional",
+      "if_else_conditional",
+      "if_elseif_conditional",
+      "if_elseif_else_conditional"
+    ],
+    "Loops": [
+      "while_loop",
+      "do_while_loop",
+      "until_loop",
+      "do_until_loop",
+      "for_loops",
+      "foreach_loops"
+    ],
+    "Iterations": [
+      "each_iteration",
+      "map_iteration",
+      "filter_iteration",
+      "fold_iteration",
+      "recursion_iteration"
+    ]
+  },
+  "control_structures": {
+    "if_conditional": {
+      "name": "If conditional",
+      "code": "
+      if(condition){
+        statements;
+      }"
+    },
+    "if_else_conditional": {
+      "name": "If/Else conditional",
+      "code": "
+      if(condition){
+        statements;
+      }
+      else{
+        statements;
+      }"
+    },
+    "if_elseif_conditional": {
+      "name": "If/ElseIf conditional",
+      "code": "
+      if(condition){
+        statements;
+      }
+      else if(condition){
+          statements;
+      }"
+    },
+    "if_elseif_else_conditional": {
+      "name": "If/ElseIf/Else conditional",
+      "code": "
+      if(condition){
+        statements;
+      }
+      else if(condition){
+          statements;
+      }
+      else{
+        statements;
+      }"
+    },
+    "while_loop": {
+      "name": "While loop",
+      "code": "
+      while(condition){
+        statements;
+      }"
+    },
+    "do_while_loop": {
+      "name": "Do/While loop",
+      "code": "
+      do{
+        statements;
+      }while(condition)"
+    },
+    "until_loop": {
+    },
+    "do_until_loop": {
+    },
+    "for_loops": {
+      "name": "For loop",
+      "code": "
+      for(initialization; condition; increment/decrement){
```

Comment:
Can you change `increment/decrement` to just `incrementor`? I want to make sure people don't get confused by the `/` in it.